### DISCUSSION TITLE
Nowe repozytorium

### DISCUSSION THREAD

**Discussion by oliwia-laskowska** [community/community#38437] [Other Feature Feedback, Questions, & Ideas]
Witam, jak tworze nowe repozytorium i pliki zostaja zaakceptowane to strony nie widac na witrynie w czy moze byc problem? W pages zmieniałam na main. Drugi problem jak umieścić pliki php żeby działały oraz w jaki sposob połączyć pliki php na githubie z baza danych?

**Comment by yoannchaudet** [comment]
Hello, GitHub Pages is a static site hosting service only. That means you cannot host there a PHP backend or connect to a database.
